tools/rna_tools/sortmerna/tool_dependencies.xml

```diff
@@ -5,8 +5,20 @@
             <actions_group>
                 <actions architecture="x86_64" os="linux">
                         <action type="download_by_url" target_filename="sortmerna-2.0.tar.gz">https://github.com/biocore/sortmerna/archive/2.0.tar.gz</action>
-                        <action type="autoconf"/>
+                        <action type="shell_command"><![CDATA[
+                            ./build.sh --prefix=$INSTALL_DIR
+                        ]]>
+                        </action>
                         <action type="change_directory">../</action>
+                        <action type="make_directory">$INSTALL_DIR/bin/</action>
+                        <action type="move_file">
+                            <source>sortmerna-2.0/sortmerna</source>
+                            <destination>$INSTALL_DIR/bin</destination>
+                        </action>
+                        <action type="move_file">
+                            <source>sortmerna-2.0/indexdb_rna</source>
+                            <destination>$INSTALL_DIR/bin</destination>
+                        </action>
```

Comment:
I tested with the 2 `<actions>` (and manually, also)
But I still have some errors :(